tracex_project/tracex/logic/utils.py

```diff
@@ -121,6 +121,7 @@ def prepare_df_for_xes_conversion(df, activity_key):
     def rename_columns(df: pd.DataFrame):
         """Rename columns in the DataFrame for better readability."""
```

Comment:
Can we make this function clearer as to when we should use it? I think it's used every time we want to display something on an HTML page or similar, and it's quite confusing that it's just called rename_columns.

Reply:
you're right. ill just use your description then 👍

Reply:
I guess this will be reworked soon either way